app/build.gradle

```diff
@@ -5,7 +5,6 @@ properties.load(project.rootProject.file('local.properties').newDataInputStream(
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "24.0.3"
```

Comment:
Is this no longer necessary, or specified elsewhere?

Reply:
Apparently it's no longer necessary (https://developer.android.com/studio/releases/build-tools.html)